modules/ROOT/pages/teradata-vantage-engine-architecture-and-concepts.adoc

```diff
@@ -0,0 +1,118 @@
+
+The major architectural components of the Teradata Vantage engine include the Parsing Engines (PEs), BYNET, Access Module Processors (AMPs), and Virtual Disks (Vdisks). Vdisks are assigned to AMPs in enterprise platforms, and to the Primary Cluster in the case of VantageCloud Lake environments. 
+
+
+image::{dir}/teradata_architecture_major_components.png[Teradata Vantage Major Architectural Components, width=75%]
+
+== Teradata Vantage Engine Architecture Components
+The Teradata Vantage engine consists of the components below:
+
+=== Parsing Engines (PE)
+When a SQL query is run in Teradata, it first reaches the Parsing Engine. The functions of the Parsing Engine are:
+
+* Manage individual user sessions (up to 120). 
+* Check if the objects used in the SQL query exist.  
+* Check if the user has required privileges against the objects used in the SQL query.  
+* Parse and optimize the SQL queries.  
+* Prepare the execution plan to execute the SQL query and passes it to the corresponding AMPs. 
+* Receives the response from the AMPs and sends it back to the requesting client. 
+
+
+=== BYNET 
+BYNET is a system that enables component communication. The BYNET system provides high-speed bi-directional broadcast, multicast, and point-to-point communication and merge functions. It performs three key functions: coordinating multi-AMP queries, reading data from multiple AMPs, regulating message flow to prevent congestion, and processing platform throughput. These functions of BYNET make Vantage highly scalable and enable Massively Parallel Processing (MPP) capabilities.  
+
+=== Parallel Database Extension (PDE)
+Parallel Database Extension (PDE) is an intermediary software layer positioned between the operating system and the Teradata Vantage database. PDE enables MPP systems to use features such as BYNET and shared disks. It facilitates the parallelism that is responsible for the speed and linear scalability of the Teradata Vantage database.  
+
+=== Access Module Processor (AMP)
+AMPs are responsible for data storage and retrieval. Each AMP is associated with its own set of Virtual Disks (Vdisks) where the data is stored, and no other AMP can access that content in line with the shared-nothing architecture. The functions of AMP are:
+
+* Access storage using Vantage’s Block File System Software  
+* Lock management 
+* Sorting rows 
+* Aggregating columns 
+* Join processing 
+* Output conversion 
+* Disk space management 
+* Accounting 
+* Recovery processing 
+
+[NOTE]
+====
+AMPs in Vantage Enterprise, and the Primary Cluster in the case of VantageCloud Lake, store data in a Block File System (BFS) format on Vdisks. AMPs in Compute Clusters and Compute Worker Nodes on Lake do not have BFS, they can only access data in object storage using the Object File System (OFS). 
```

Comment:
If we say VantageCloud Lake, then we should use the full name for VantageCloud Enterprise.
Also, why does this callout mention only VantageCloud editions? Why not VantageCore IntelilFlex and VantageCore VMware?

Reply:
Yeah spent some time today to understand this more. I have added about VantageCloud Enterprise, VantageCore Intelliflex and VantageCore VMware too.